README.md

```diff
@@ -0,0 +1,37 @@
+Wybrałem spring boota ponieważ wydawał mi się najprostrzy w konfiguracji i obsłudze.  
```

Comment:
Spring Boot to zazwyczaj dobry wybór :+1:

Reply:
Tak w kwestii eksperymentu myślowego: Czy uważasz, że taka aplikacja i wybór technologii obsłużyłoby np. 800 mln SMS'ów dziennie? :thinking:

Reply:
Myślę że aplikacja wymaga dopieszczenia jeśli miałaby przyjmować taką ilość requestów dziennie :)
Jednak, sam wybór technologii mógłby podołać. 
Napisałem wszystko w miarę moich umiejętności, bardzo się ciesze się doceniam że przeanalizowałeś moje rozwiązanie. Super, że wyłapałeś tak dużo sugestii i zmian które mógłbym wprowadzić, bo to są istotne sprawy o których przy pisaniu kodu nie pomyślałem. Także bardzo dziękuję za analizę i będę starał się te dobre praktyki wdrażać na przyszłość  :)